pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -413,14 +417,22 @@ func parseAndSendSyslogEntries(ctx *loggerContext) {
 					logSource = splitArr[1]
 				}
 			}
+		} else if logInfo.Containername != "" {
+			logSource = logInfo.Containername
```

Comment:
We should at least document that the source is used in this way. I don't logging if logging.md specifies which fields are used for what. Good to check with @gkodali-zededa

Reply:
If these are considered app logs, they are put into a different database. Developers cannot see these  logs from Kibana.

Reply:
Right.